server/pbench/bin/pbench-server-prep-version-001

```diff
@@ -0,0 +1,182 @@
+#! /bin/bash
+
```

Comment:
Can we add a short description of that this script does here?

Reply:
Yes.